tests/common/include/tests/common/tools/intersectors/helix_cylinder_intersector.hpp

```diff
@@ -69,6 +70,8 @@ struct helix_cylinder_intersector {
 
         // Starting point on the helix for the Newton iteration
         // The mask is a cylinder -> it provides its radius as the first value
+        // @TODO: Can it be general enum class rather than using specific
```

Comment:
Which other shape would you want to use? Even if cylinder3D could be intersected, it has two different r values and would likely need its own intersector anyway

Reply:
If you think it is the best then let's leave it as it is. I was not sure if it was meant to include `cylinder2D.hpp`

Reply:
This kind of situation was the intent for the enum in the cylinder shape.
Yes, I think, I just forgot to include it :D